components/icon/demo/basic.md

```diff
@@ -7,8 +7,12 @@ title:
 
 ## zh-CN
 
-使用 `<span ></span>` 标签声明组件，指定图标对应的 `nzType` 属性。可以通过 `nzTheme` 属性来设置不同的主题风格的图标，也可以通过设置 `nzSpin` 属性来实现动画旋转效果。对旧的 API `<i class="anticon"></i>` 兼容。
+推荐使用 `<nz-icon>` 而不是 `<span>` 标签来声明组件。
```

Comment:
I think we need to add a note informing the user that `<nz-icon>` selector is supported since `v19`, and both `<i>` and `<span>` are still supported.